lib/wasm/tests/testsuite.rs

```diff
@@ -23,6 +23,14 @@ fn testsuite() {
     let mut paths: Vec<_> = fs::read_dir("../../wasmtests")
         .unwrap()
         .map(|r| r.unwrap())
+        .filter(|p| {
+            if let Some(stem) = p.path().file_stem() {
+                if let Some(stemstr) = stem.to_str() {
+                    return !stemstr.starts_with(".");
```

Comment:
Please add a comment mentioning the motivation.

Reply:
Amended that commit with a comment